cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -854,3 +854,46 @@ def _revert():
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.credentials.get_text_of("Vault") == vault_creds.name
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(automates=[BZ(1734904)])
+def test_ansible_service_ansible_galaxy_role(
+    appliance,
```

Comment:
please don't format arguments in such way. we already have formatting stile for functions/methods. please stick to it.

Reply:
Okay, Updating it.

Reply:
Updated.